packages/react-native-skia/components/image/blurRadius.js

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { View, AppRegistry, Image, Text } from 'react-native';
+var resizemodes = [
+    "center",
+    "cover",
+    "contain",
+    "stretch",
+    "repeat",
+];
+var blurRadius =[0,-1,-3,5,10,50,100,200,2000]
+var useCases =9;
+var timerValue=3000;
+
+const SimpleViewApp = React.Node = () => {
+
+const [UseCaseCount, toggleState] = useState(0);
+  setTimeout(()=>{
+     if(UseCaseCount == useCases ) toggleState(0)
+     else toggleState(UseCaseCount+1);    
+  }, timerValue)
+
+  return (
```

Comment:
based on the changes , below cases need to be verified in the test app .please confirm 
a) blur radius + no background color + shadow 
b) blur radius + no background color + no shadow 
c) blur radius + background color + shadow 
d) blur radius + background color + no shadow 
e) all the above with no blur radius 
f) all the above with no blur radius + repeat resize mode